lib/modules/versioning/distro.ts

```diff
@@ -127,8 +129,8 @@ export class DistroInfo {
     }
 
     if (end) {
-      const now = DateTime.now();
-      const eol = DateTime.fromISO(end);
+      const now = DateTime.now().toUTC();
+      const eol = DateTime.fromISO(end).toUTC();
```

Comment:
```suggestion
      const eol = DateTime.fromISO(end, { zone: 'utc' });
```

One less function call 🙃

Reply:
https://moment.github.io/luxon/#/zones